superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -248,10 +247,6 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     const { id, ...update } = db || {};
     if (db?.id) {
-      if (db.sqlalchemy_uri) {
```

Comment:
@eschutho shouldnt we delete this since `sqlalchemy_uri` will always be available via GET /databases/<id>

Reply:
there won't be a sqlalchemy_uri if you are creating from scratch and using the form view.